src/Maker/MakeRegistrationForm.php

```diff
@@ -66,32 +69,36 @@ public static function getCommandName(): string
         return 'make:registration-form';
     }
 
-    public function configureCommand(Command $command, InputConfiguration $inputConf)
+    public function configureCommand(Command $command, InputConfiguration $inputConf): void
     {
         $command
             ->setDescription('Creates a new registration form system')
             ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeRegistrationForm.txt'))
         ;
+
+        $arguments = [
+            'user-class' => true,
+            'username-field' => true,
+            'password-field' => true,
+            'will-verify-email' => true,
+            'id-getter' => false,
+            'email-getter' => false,
+            'from-email-address' => false,
+            'from-email-name' => false,
+            'auto-login-authenticator' => false,
+            'firewall-name' => false,
+            'redirect-route-name' => false,
+            'add-unique-entity-constraint' => false,
+        ];
+
+        foreach ($arguments as $name => $required) {
+            $argument = new MakerArgument($name, null, $required);
```

Comment:
We could also call `$this->arguments->createArgument($name, null, $required);` instead of creating a new `MakerArgument` then using `$this->arguments->addArgument(MakerArgument)` Either way provides the same result...